tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -1040,6 +1041,48 @@ test.describe( 'Nested Tabs tests @nested-tabs', () => {
 
 		await cleanup( wpAdmin );
 	} );
+
+	test( 'Verify that the active tab titles are visible on the accordion view @latest', async ( { page }, testInfo ) => {
```

Comment:
@hein-obox As discussed, can we not make this part of a current test? To save some time? If it does not make sense for this test, ignore this comment :-)

Reply:
@louiswol94 I have updated it.
I have created a separate file for each test.
I think that might help for readability, but it also has issues, because you can't see the settings of others tests. 

This isn't so easy...

I'd be keen to huddle about this.

Reply:
@hein-obox Let's leave it for now. @TzviRabinovitch said this morning in the call that we should not do this anymore.